### PR TITLE
CAD-425 | journal scribe:  set a syslog identifier

### PR DESCRIPTION
**Pull request by deepfire**
This is required to properly drive the syslog identifier field -- which is otherwise empty, resulting in un-attributed messages like:

```
Jan 20 19:24:40 andromedae [18343]: System started at SystemStart {getSystemStart = 2020-01-09 16:50:01 UTC}
Jan 20 19:24:40 andromedae [18343]: **************************************
                                    I am Node NodeAddress {naHostAddress = NodeHostAddress {getAddress = Nothing}, naPort = 1234} Id: 0
                                    My producers are [RemoteAddress {raAddress = "127.0.0.1", raPort = 3001, raValency = 1},RemoteAddress {raAddress = "127.0.0.1", raPort = 3002, raValency = 1}]
                                    **************************************
```

description
-----------

- [ ] describe solution here ..

checklist
---------

- [x] compiles (`cabal new-clean; cabal new-build`)
- [x] tests run successfully (`cabal new-test`)
- [ ] documentation added and created (`cd docs; nix-shell --run make`)
- [ ] link to an issue
- [ ] link to an epic
- [ ] add estimate points
- [ ] add milestone (the same as the linked issue)

### CODE REVIEW

File: plugins/scribe-systemd/src/Cardano/BM/Scribe/Systemd.lhs [hunk 1]
```diff
@@ -58,7 +58,7 @@ plugin :: (IsEffectuator s a, ToJSON a, FromJSON a)
        => Configuration -> Trace IO a -> s a -> IO (Plugin a)
 plugin _ _ _ =
     ScribePlugin
-               <$> mkJournalScribe
+               <$> mkJournalScribe "cardano"
```
Comment (CodiePP) [inline plugins/scribe-systemd/src/Cardano/BM/Scribe/Systemd.lhs]: what if the identifier can be passed in as an argument?

Comment (deepfire) [reply plugins/scribe-systemd/src/Cardano/BM/Scribe/Systemd.lhs]: @CodiePP, done.

**Comment by CodiePP** [comment]
bors -r

**Pull request merged**